src/context-parser-handlebars.js

```diff
@@ -392,7 +392,7 @@ ContextParserHandlebars.prototype.analyzeAst = function(ast, contextParser, char
                 node.type === handlebarsUtils.RAW_EXPRESSION) {
 
                 // lookupStateForHandlebarsOpenBraceChar from current state before handle it
-                parser.setCurrentState(ContextParserHandlebars.lookupStateForHandlebarsOpenBraceChar[parser.state]);
+                parser.state = ContextParserHandlebars.lookupStateForHandlebarsOpenBraceChar[parser.state];
```

Comment:
got it. but why do we not using the api `setCurrentState()`? instead of assigning directly to the internal var?